content/docs/user-guide/gems/reference/aws/aws-gamelift/cpp-api.md

```diff
@@ -130,22 +129,27 @@ As the default behavior, when the last player leaves the game session, GameLift
 
 ## Server notifications
 
-The AWS GameLift Gem notifies the GameLift service that your server process is ready to host game sessions, handle requests, and take connections.
+You must notify Amazon GameLift service that your server process is ready to host game sessions, handle requests, and take connections.
+
+To send a notification that your server process is ready, use `AWSGameLiftServerRequestBus::Events::NotifyGameLiftProcessReady()`.
+Put this call in a proper place on your server side, like server system component activate step, and call it after you connect your project to `AzFramework::SessionNotificationBus`.
+
+```cpp
+AWSGameLift::AWSGameLiftServerRequestBus::Broadcast(&AWSGameLift::AWSGameLiftServerRequestBus::Events::NotifyGameLiftProcessReady);
+```
 
-When your session is active, it broadcasts the following notifications to the GameLift server. You can program how your session responds to these notifications.  
+Once your session is created, it broadcasts the following notifications through `AzFramework::SessionNotificationBus` on server side. You can program how your session responds to these notifications.  
```

Comment:
nit: "Once the game session is created, the a notification is broadcast through... You can then listen to the notification to {insert example of what customer should do here}"